app/src/main/res/drawable/ic_accessible_black_24dp.xml

```diff
@@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FF000000"
```

Comment:
You can replace the hex codes with @string/colorPrimary, which will make it the same blue as the navbar as well as allow easy changes to colour scheme later